### ISSUE TITLE
UI: all bold font style

### ISSUE THREAD

**Issue by ghmon** [git-cola/git-cola#1516]
On small displays, or for people with visual impairments an "all bold" font style could be helpful.

I modified cola/app.py:

        self._install_style(gui_theme)
        # set bold font globally
        self._app.setStyleSheet("QWidget { font-weight: bold; }")

For sure this should be configurable.

**Comment by davvid** [comment]
There's now a "Bold all fonts" option in the Appearance settings. Thanks for the accessibility sug.
